web-app/packages/lib/src/modules/instance/store.ts

```diff
@@ -30,14 +31,22 @@ export const useInstanceStore = defineStore('instanceModule', {
 
   getters: {
     /**
-     * Checks if global roles are enabled based on the config data.
-     * Returns true if any of the global role flags are truthy, false otherwise.
+     * Retrieves the current global role from the instance configuration data.
+     *
+     * The global role is determined by checking the `global_read`, `global_write`, and `global_admin` flags in the configuration data.
+     * If none of the flags are set, `undefined` is returned, otherwise the index of the first set flag is returned as the global role.
+     *
+     * @param state - The current instance state.
+     * @returns The current global role, or `undefined` if no global role is set.
      */
-    globalRolesEnabled(state): boolean {
+    currentGlobalRole(state): GlobalRole | undefined {
```

Comment:
Just a note: does it matter that in EE and cloud there is not such thing? What would happen if backend returned such flag?

Reply:
Components in other environments are not connected to this variable.